do.py

```diff
@@ -147,6 +147,27 @@ def test():
         else:
             print("Coverage thresold[{0}] is achieved[{1}]".format(coverage_threshold, result))
 
+    go_coverage_threshold = 35
+    # TODO: not able to run the test from main directory
+    os.chdir("pkg")
+    try:
+        run(["go test ./... -v -coverprofile coverage.txt | tee coverage.out"])
+    except:
+        os.chdir("..")
```

Comment:
do this as part of "finally". don't repeat twice

Reply:
updated